nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -15,6 +15,7 @@
       <input_port name="server_timeout">Server timeout</input_port>
       <input_port name="disable_collision_checks">Disable collision checking</input_port>
       <output_port name="error_code_id">"Back up error code"</output_port>
+      <output_port name="error_msg">"Back up error message"</output_port>
```

Comment:
A couple of BT nodes are missing. I count 10x header files updated (missing wait) and only 9 changes here. So I think there's 2 missing.